frontend/src/v5/ui/components/shared/dashboardFooter/dashboardFooter.styles.tsx

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
Same comment about file extension

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/43547703/177329976-027418e9-1f5e-4cd3-816b-b0563998bd2e.png)
It still shows as `.tsx.`

Reply:
Have ammended

Reply:
![this-is-getting-out-of-hand-now-there-are-two](https://user-images.githubusercontent.com/42032433/178031662-968f27fe-f5d2-469b-a8d9-ccb65271a184.gif)